### ISSUE TITLE
EXC_BAD_ACCESS when numberOfCellsInStreamView < numberOfColumnsInStreamView

### ISSUE THREAD

**Issue by taketo1024** [ekohe/EKStreamView#10]
I get EXC_BAD_ACCESS on condition: numberOfCellsInStreamView < numberOfColumnsInStreamView. This happens when the data to be displayed is still empty (before being loaded).

You can repro this error by modifying the sample code as:

``` objc
- (NSInteger)numberOfCellsInStreamView:(EKStreamView *)streamView
{
    return 5;
}

- (NSInteger)numberOfColumnsInStreamView:(EKStreamView *)streamView
{
    return 10;
}
```

**Comment by Xenofex** [comment]
OK It's fixed by 2855c68fc000287287e32427c1030bf055d9d25e. Thank you and @cyndibaby905

**Comment by cyndibaby905** [comment]
@Xenofex  With pleasure:)
